application/gqlgen/schema.graphql

```diff
@@ -312,6 +312,7 @@ input LocationInput {
 input UpdatePostStatusInput {
     id: ID!
     status: PostStatus!
+    userID: ID
```

Comment:
I wonder if the COMMITTED to ACCEPTED transition would be better as a new mutation since it's the only one that needs a User ID.

Reply:
Phillip was wondering about removing the possibility of a committed status.  That would simplify the code a bit and then make it necessary to add a new mutation with the new potential provider.
Then also, all the Posts we currently consider COMMITTED would end up being OPEN, allowing request for a list of Posts to stay what it is now.

Reply:
So that would do away with the whole idea of "committed" and just leave that process up to user-to-user communication. What if the requester just chooses from the users that have an open thread? No new tables and only one new mutation to select the provider.

Reply:
I think we still need to keep track of potential providers in the API.  I doubt if the UI guys would want to do that.

Reply:
I wasn't suggesting that the UI kept track of potential providers. My thought was that merely opening a thread qualifies a user as a "potential provider". That only means that the requester would see an "accept" button on that thread. It would be up to the human interaction to determine whether that makes sense for any given user.

Reply:
One reason for the API to keep track of the explicit potential providers is so it can notify the rejected ones once the requester has accepted a different offer.

Reply:
I hadn't thought of that. Still, that could be either implied by lack of communication or left up to the requester to let others know it was handled. Not ideal, but simple.